### DISCUSSION TITLE
How to insert in AUTOCOMPLETE  3 parameters with comma delimiter?

### DISCUSSION THREAD

**Discussion by rafa4mat** [WinEdt-Team/WinEdt-Support#79] [1.1 General Help and Discussion]
I tried with 

```
OBJECT="\definecolor{name color}{model}{'bullet','bullet','bullet'}"
MACRO="Exe('%b\Macros\Auto Complete\TeX_Arg.edt');"
```
The problem is that I would like to obtain 
`\definecolor{'bullet'}{'bullet'}{'bullet','bullet','bullet'}`
but I obtain 
`\definecolor{'bullet'}{'bullet'}{'bullet'} with ONLY one 'bullet' instead of {'bullet','bullet','bullet'} in the final parameter`

I supose that I have to use another edt file instead of TeX_Arg.edit

**Comment by Alex-Simonic** [comment]
Use:
```
    OBJECT="\definecolor{name color}{model}{,,}"
    MACRO=""
```
and see what happens... If you start with two bullets followed by a string and a single bullet the Next Bullet (`Ctrl+Space`) will select the whole thing. Introducing an option to select the model from the list of predefined values (RGB,...,gray) would require a custom macro (using MUI). It is probably too much work to be worth the effort unless somebody (other than me) feels like spending an hour or so implementing it. 

TeX_Arg.edt replaces any string inside {...} with a single bullet. That's why it cannot be used for this... But an empty MACRO will do just fine in this case.

**Reply by rafa4mat** [reply]
Thank you, Alex. Works fine 

`'bullet' 'bullet' word 'bullet'`
